components/activities/activities-tab/activities-tab.tsx

```diff
@@ -41,29 +35,19 @@ export function ActivitiesTabManager<
   ActivityType extends ActivityListItemTypes[SourceType],
 >(props: ActivitiesTabManagerProps<ActivityListItemTypes>) {
   useStyleSheet(styleSheet);
+
   const orgUserWithRoles$ = useOrgUserWithRoles$();
   const { pushDialog } = useDialog();
   const activityListItems = props.activityListItems ?? DEFAULT_LIST_ITEMS;
 
-  const activityAlertConnection$ = usePollingActivityAlertConnection$<ActivityType, SourceType>(
-    { interval: 2000, limit: 20 },
-  );
-
-  const activeActivities = useSelector(() =>
-    activityAlertConnection$.data?.get()?.alertConnection.nodes.filter((activity) =>
-      activity?.activity
-    ).filter(isActiveActivity) || []
-  );
-
-  // HACK - this is a hack to only open the active prediction
-  // remove once we've cleaned up the prediction component so we can
-  // render past prediction results
-  const firstPrediction = activeActivities.find((activity) => isPrediction(activity.activity));
+  const { activityAlertConnection$, reexecuteActivityConnectionQuery } =
+    usePollingActivityAlertConnection$<ActivityType, SourceType>(
+      { interval: 2000, limit: 20 },
```

Comment:
i'd probably still pull these out into comments, even with the named properties. but i don't feel strongly about that at all